### DISCUSSION TITLE
Standard quality of streams

### DISCUSSION THREAD

**Discussion by T1JNES** [f1multiviewer/issue-tracker#391] [Feature requests]
I would like to be able to set the standard quality of streams in my settings so it wont go on auto but i can set it to 1080p standard 😄

**Comment by jeppeolesen** [comment]
Hey @T1JNES 

Just so I understand you correctly: You'd like an option in MultiViewer to set the standard quality of all the streams you open, instead of it being defaulted to "Auto"?
